### PR TITLE
Remove use of old FileInPath class

### PR DESCRIPTION
**Pull request by Dr15Jones**
This was the last use of the old (non edm) FileInPath class. It
is unclear why this code was not using edm::FileInPath, since
it essentially is doing the same work, but that avoidance was
kept in this version.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

File: PhysicsTools/Utilities/interface/RootMinuitCommands.h [hunk 1]
```diff
@@ -126,27 +126,34 @@ namespace fit {
     using namespace std;
```
Comment (davidlange6) [inline PhysicsTools/Utilities/interface/RootMinuitCommands.h]: hi @Dr15Jones - while noticing it - could you get rid of this using std?

Comment (Dr15Jones) [inline PhysicsTools/Utilities/interface/RootMinuitCommands.h]: I can, however that using is within the scope of a function which is allowed by our coding conventions.

Comment (davidlange6) [inline PhysicsTools/Utilities/interface/RootMinuitCommands.h]: whoops, I missed that - never mind..

**Comment by davidlange6** [comment]
merge

**Pull request merged**